packages/expo-random/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ### 🛠 Breaking changes
 
+- Deprecate the library in favor of expo-crypto. ([#20217](https://github.com/expo/expo/pull/20217) by [@aleqsio](https://github.com/aleqsio))
```

Comment:
Move this up to `Unpublished` instead of version `13.0.0` that is already released 😉 
Also, it might be better to put it under `### ⚠️ Notices` category instead of breaking changes (this category is supported by our tools, it's just not added to changelogs by default)